trl/trainer/grpo_trainer.py

```diff
@@ -905,6 +907,8 @@ def _generate_and_score_completions(
                         "reward": rewards.tolist(),
                     }
                     df = pd.DataFrame(table)
+                    if self.num_completions_to_log is not None:
```

Comment:
For consistency, I've enabled subsampling here but we could skip it for WandB since it doesn't really matter if there's a lot of completions AFAIK

Reply:
I think for wandb we should just keep everything? It is more of an issue of the logs being spammed

Reply:
Sounds good, I'll revert